### ISSUE TITLE
Automate new test creation

### ISSUE THREAD

**Issue by 0xLucqs** [software-mansion/cairo-lint#2]
Write a script to automate the test creation process. The steps are the following:

To create a new test for a lint you need to create a file
in the [test_files folder](./crates/unretardify-core/tests/test_files/) and should be named as your lint. The file
should have this format:

```txt
//! > Test name

//! > cairo_code
fn main() {
    let a: Option<felt252> = Option::Some(1);
}
```

Then in the [test file](crates/unretardify-core/tests/tests.rs) declare your lint like so:

```rs
test_file!(if_let, "Test name");
```

**Comment by coxmars** [comment]
Hi @0xLucqs,

I've implemented the "Automate new test creation" feature, allowing test files to be generated in two ways: by passing a lint name as an argument or by prompting the user if no argument is provided. 

The script is working but I need to fix and improve more things so I'd love to continue contributing to this project. Could you please assign this issue to me? 🫡

![image](https://github.com/user-attachments/assets/1527e95c-5d07-43b2-bbd7-86fbf26d915f)

**Comment by 0xLucqs** [comment]
Hey @coxmars thanks for doing that. Though I think that you would either need to create a new cli like `cairo-lint-dev` or maybe a bash script for that up to you.

**Comment by 0xLucqs** [comment]
Fixed by #25
